hasher-matcher-actioner/webapp/README.md

```diff
@@ -2,7 +2,16 @@
 
 # Getting started with the hasher-matcher-actioner webapp
 
-This project requires [npm](https://www.npmjs.com/get-npm) be installed. Once installed  `npm install` from within this directory should enable usage of the scripts referenced below.
+This project requires [npm](https://www.npmjs.com/get-npm) be installed. Once installed `npm install` from within this directory should enable usage of the scripts referenced below.
+
+To run, build or test you need to create a .env file in the webapp directory. One is created for you when you terraform apply, and it should look something like what you see in .env.example.
+
+To sign in (whether running from localhost, the s3 bucket url, or the cloudfront distribution) you need a user. Take these steps to create a user after you've run terraform apply:
```

Comment:
❤️, think is it worth updating CONTRIBUTING.md with something like "For the UI/Webapp components see `webapp/README` ?